build_examples.sh

```diff
@@ -4,9 +4,11 @@ mkdir -p bin
 
 echo "Building to bin/ folder"
 
-#CGO required to build example relying on crypto11 dependency.
-find . -name "*.go" ! -name "*pkcs11*.go" -print0 | CGO_ENABLED=0 xargs -0 -n1 go build
-find . -name "*pkcs11*.go" -print0 | CGO_ENABLED=1 xargs -0 -n1 go build
+# CGO required to build example relying on crypto11 and imagick dependency.
+find . -name "*.go" ! -name "*_cgo.go" -print0 | CGO_ENABLED=0 xargs -0 -n1 go build -o bin/
+find . -name "*pkcs11*_cgo.go" -print0 | CGO_ENABLED=1 xargs -0 -n1 go build
 
-mv pdf_* fdf_* jbig2_* bin/
+# Custom encoder example need another CGO flags to build imagick lib.
+find . -name "*custom_encoder*_cgo.go" -print0 | CGO_ENABLED=1 CGO_CFLAGS_ALLOW='-Xpreprocessor' xargs -0 -n1 go build
```

Comment:
Is the -Xpreprocessor needed?  or can we reduce this to only 1 step for all cgo related files? i.e.
```
find . -name "*_cgo.go" -print0 | CGO_ENABLED=1 xargs -0 -n1 go build
```

Reply:
Maybe add to the project README also:
> While the majority of examples are fully in pure Go, there are a few examples that demonstrate additional functionality that requires CGO and external dependencies. Those examples are clarified by filename suffix "_cgo.go".

Reply:
yest the flag is needed by imagick